packages/deploy-tooling/test/unit/cli/index.test.ts

```diff
@@ -85,19 +108,21 @@ describe('cli', () => {
             expect(mockedUi5RepoService.undeploy).toBeCalled();
         });
 
-        test('unsuccessful undeploy, help options is shown if no parameters are passed in', async () => {
-            // Dont exit the jest process
-            const mockExit = jest.spyOn(process, 'exit').mockImplementation((number) => {
-                throw new Error('process.exit: ' + number);
-            });
-            const errorMock = jest.spyOn(Command.prototype, 'error').mockImplementation();
-            const helpMock = jest.spyOn(Command.prototype, 'help');
-            process.argv = ['node', 'test'];
+        test('successful undeploy with environment variable and no config file', async () => {
```

Comment:
The test description makes me a little uncomfortable, it only successfully _calls_ `undeploy`, doesn't it?